services/src/handlers/datasets.rs

```diff
@@ -144,13 +183,69 @@ async fn get_dataset_handler<C: Context>(
     Ok(web::Json(dataset))
 }
 
+/// Creates a new dataset using files available on the volumes.
+/// Requires an admin session.
+#[utoipa::path(
+    tag = "Datasets",
+    post,
+    // TODO: "public" doesn't really make sense, because there are no private datasets. But this is more consistent with pro version
```

Comment:
Ah okay, you also saw it yourself that this handler can only be temporary.

Reply:
68ba46b28acc48bfeb8fad8ef79447178e18c62b